composer.json

```diff
@@ -30,7 +30,7 @@
         "symfony/yaml": "~2.8|~3.0|~4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8.26|^5.0",
+        "phpunit/phpunit": "^4.8.35|^5.7|^6.5",
```

Comment:
Is it easier at some point if we would stop supporting completely outdated php version 5.4 and e.g. use 5.6+ instead?
Then we could drop 4.8 here probably at least.

Reply:
It's up to you. I've made everything to support all `PHP` and `PHPUnit` versions. But, I can open other PR dropping PHP `5.4` and `5.5` support :man_technologist:

Reply:
For now its cool then :)

Reply:
Gonna work on stop supporting for PHP `5.4` and `5.5` versions soo.

Reply:
I meant you dont have to. If all works fine :)

Reply:
Ahh, okay, sorry :sweat_smile: